packages/cli/lib/fs.js

```diff
@@ -0,0 +1,10 @@
+const { readFile, writeFile, copyFile, stat, statSync } = require('fs');
+const { promisify } = require('util');
+
+module.exports = {
+	copyFile: promisify(copyFile),
+	readFile: promisify(readFile),
+	writeFile: promisify(writeFile),
+	stat: promisify(stat),
+	statSync,
+};
```

Comment:
was kinda annoyed by the abandoned dependency and we'll get native promise support 🔜™